alica_engine/include/engine/collections/RobotProperties.h

```diff
@@ -12,12 +12,13 @@ namespace alica
 {
 
 class AlicaEngine;
```

Comment:
Declaration of AlicaEngine can be removed now

Reply:
+1